docs/src/conf.py

```diff
@@ -89,7 +89,9 @@ def autolog(message):
 version = get_version("scitools-iris")
 if version.endswith("+dirty"):
     version = version[: -len("+dirty")]
+release = version
```

Comment:
Reinstating this from #4855, as it's part of the basic `sphinx` project information, see [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-release)